sdk/monitor/monitor-opentelemetry/src/utils/statsbeat.ts

```diff
@@ -14,6 +14,24 @@ let instance: StatsbeatConfiguration;
 class StatsbeatConfiguration {
   // Initial Statsbeat options
   private currentStatsbeatOptions: StatsbeatOptions = {};
+  private initializedByShim = false;
+
+  constructor() {
+    // Check for shim initialization upon construction
+    try {
+      if (
+        JSON.parse(process.env[AZURE_MONITOR_STATSBEAT_FEATURES] || "{}").feature &
```

Comment:
This code feel weird, are you not doing exact same thing of parsing the environment variable and calling setStatsbeatFeaures somewhere else? can you reuse that?